Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM_2nd.cpp

```diff
@@ -67,6 +69,8 @@ void FiniteDifferenceSolver::MacroscopicEvolveHMCartesian_2nd(
     amrex::GpuArray<int, 3> const& mag_Ms_stag    = macroscopic_properties->mag_Ms_IndexType;
     amrex::GpuArray<int, 3> const& mag_alpha_stag = macroscopic_properties->mag_alpha_IndexType;
     amrex::GpuArray<int, 3> const& mag_gamma_stag = macroscopic_properties->mag_gamma_IndexType;
+    amrex::GpuArray<int, 3> const& mag_exch_stag  = macroscopic_properties->mag_exch_IndexType;
+    amrex::GpuArray<int, 3> const& mag_ani_stag   = macroscopic_properties->mag_ani_IndexType;
```

Comment:
Its best to write out "exchange" and "anistotropic" even though they may seem long. Its easier to read, and follow and it will also be caught when we "grep" while debugging

Reply:
Sure.